index.bs

```diff
@@ -1600,6 +1598,8 @@ interface XRWebGLLayer {
 };
 </pre>
 
+Each {{XRWebGLLayer}} has a <dfn for="XRWebGLLayer">context</dfn> object, initially <code>null</code>, which is an instance of either a {{WebGLRenderingContext}} or a {{WebGL2RenderingContext}}.
```

Comment:
You can just say that this is an `{{XRWebGLRenderingContext}}` (but you don't have to, this is fine too)

Reply:
I think this is actually slightly clearer in this case, since it avoids the need to cross-reference with other parts of the spec. Since it's not a huge difference in the size of the text, nor is it something that gets repeated a lot elsewhere, I'm good with it.